lib/librte_hash/rte_cuckoo_hash.c

```diff
@@ -2659,3 +2659,32 @@ rte_hash_iterate(const struct rte_hash *h, const void **key, void **data, uint32
 	(*next)++;
 	return position - 1;
 }
+
+void
+rte_hash_prefetch_buckets(const struct rte_hash *h, hash_sig_t sig)
```

Comment:
Can we make this function inline? If so make it and its sister.

Reply:
No, as this function calls inline functions only available in ``rte_cuckoo_hash.c``.